### ISSUE TITLE
rewrite in rust

### ISSUE THREAD

**Issue by ameerabdallah** [codediodeio/rektor-db#11]
rewrite in rust

**Comment by urosjarc** [comment]
@ameerabdallah This project already works perfectly in scratch... Can you write your usecases why you want rewrite it in rust?

**Comment by Preet-Sojitra** [comment]
> @ameerabdallah This project already works perfectly in scratch... Can you write your usecases why you want rewrite it in rust?

because rust is blazingly fast 🚀

**Comment by ameerabdallah** [comment]
> @ameerabdallah This project already works perfectly in scratch... Can you write your usecases why you want rewrite it in rust?

rust

**Comment by Dark-Peace** [comment]
I think we should create our own Rektor language. That's the only way the genius of this project can truly develop itself

**Comment by matveitolochko** [comment]
of course, check out my Fork: https://github.com/AtomicGamer9523/rektor-db

**Comment by codemeasandwich** [comment]
You should use a vector language like brainfuk to write a vector database Hahaha 🤣

**Comment by GGIEnrike** [comment]
> You should use a vector language like brainfuk to write a vector database Hahaha 🤣

sounds good

can you make a PR?

**Comment by matveitolochko** [comment]
@GGIEnrike did
